### PR TITLE
Makes eco_labels plural in vehicle_types

### PR DESCRIPTION
**Pull request by richfab**
#### **What problem does your proposal solve? Please begin with the relevant issue number. If there is no existing issue, please also describe alternative solutions you have considered.**
As discussed in https://github.com/MobilityData/gbfs/pull/542, the field `eco_labels` in [vehicle_types.json](https://github.com/MobilityData/gbfs/blob/master/gbfs.md#vehicle_typesjson) is an Array of objects so the field name should be plural.

#### **What is the proposal?**
Change the field name from `eco_label` --> `eco_labels` (plural) in [vehicle_types.json](https://github.com/MobilityData/gbfs/blob/master/gbfs.md#vehicle_typesjson).

#### **Is this a breaking change?**
- [x] Yes 
- [ ] No
- [ ] Unsure

#### **Which files are affected by this change?**
[vehicle_types.json](https://github.com/MobilityData/gbfs/blob/master/gbfs.md#vehicle_typesjson)

### CODE REVIEW

File: gbfs.md [hunk 1]
```diff
@@ -515,9 +515,9 @@ Field Name | REQUIRED | Type | Defines
...
+\- `eco_labels`<br/>*(added in v2.3)* | OPTIONAL | Array | Vehicle air quality certificate. Official anti-pollution certificate, based on the information on the vehicle's registration certificate, attesting to its level of pollutant emissions based on a defined standard. In Europe, for example, it is the European emission standard. The aim of this measure is to encourage the use of the least polluting vehicles by allowing them to drive during pollution peaks or in low emission zones.<br /><br />Each element in the array is an object with the keys below.
```
Comment (richfab) [inline gbfs.md]: As of v3.0-RC2

```suggestion
\- `eco_labels`<br/>*(as of v3.0-RC2)* | OPTIONAL | Array | Vehicle air quality certificate. Official anti-pollution certificate, based on the information on the vehicle's registration certificate, attesting to its level of pollutant emissions based on a defined standard. In Europe, for example, it is the European emission standard. The aim of this measure is to encourage the use of the least polluting vehicles by allowing them to drive during pollution peaks or in low emission zones.<br /><br />Each element in the array is an object with the keys below.
```

**Comment by richfab** [comment]
I hereby call a vote on this proposal. Voting will be open for 10 full calendar days until 11:59PM UTC on Monday, November 13th.
Please vote for or against the proposal, and include the organization for which you are voting in your comment.
Please note if you can commit to implementing the proposal.

**Comment by richfab** [comment]
Tagging recent contributors for visibility: @simonsolnes, @PierrickP, @futuretap, @testower, @cmonagle, @ezmckinn, @Cj-Malone, @tdelmas, @AntoineAugusti, @ArashMansouri. Thank you!

**Comment by testower** [comment]
Entur votes in favor of this proposal

**Comment by jkurzanski** [comment]
At Vulog we're ok with this

**Comment by PierrickP** [comment]
Fluctuo: We vote for ! ✅

**Comment by cmonagle** [comment]
+1 from Transit

**Comment by richfab** [comment]
Voting on this PR closes in 3 calendar days. Please vote for or against the proposal, and include the organization for which you are voting in your comment. Please note if you can commit to implementing the proposal.

**Comment by richfab** [comment]
This vote has now closed, and it passes!

Votes in favor:
Entur (consumer)
Vulog (producer)
Fluctuo (consumer)
Transit app (consumer)
 
There were no votes against.

This change will be added to v3.0-RC2, which will be released in the coming days.

**Review by josee-sabourin — APPROVED** [review]

**Pull request merged**